JS/lib/cards.js

```diff
@@ -0,0 +1,142 @@
+//   <section id="question-card0">
+//   <div class="question-card__total">
+//     <h2>QUESTION X</h2>
+
+//     <p>Can you die without perishing ?</p>
+
+//     <button class="question-card__answer">Show Answer</button>
+
+//     <i class="fas fa-bookmark question-card__bookmark-non"></i>
+//     <p class="answer">
+//       When your body leaves your soul, your soul will be forever
+//     </p>
+//   </div>
+// </section>
+
+export function createCard(card) {
+  const cardSection = document.createElement('section')
+
+  const headlineFrame = document.createElement('div')
+  cardSection.append(headlineFrame)
+  headlineFrame.classList.add('question-card__total-headline-frame')
+
+  const headLine = document.createElement('h2')
+  headLine.textContent = 'QUESTION'
+  headlineFrame.append(headLine)
+  headLine.classList.add('question-card__total-headline')
```

Comment:
Do you need the `div` around the `h2`? Maybe the `h2` is enough?

Reply:
h2 didn't change, hence my change. but I get it ;)